package.json

```diff
@@ -68,17 +69,17 @@
     "get-pkg-repo": "4.1.1",
     "kcd-scripts": "11.2.2",
     "prettier": "2.5.1",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-test-renderer": "17.0.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-test-renderer": "^18.2.0",
     "ts-node": "10.5.0",
     "typescript": "4.5.5"
   },
   "peerDependencies": {
-    "@types/react": "^16.9.0 || ^17.0.0",
-    "react": "^16.9.0 || ^17.0.0",
-    "react-dom": "^16.9.0 || ^17.0.0",
-    "react-test-renderer": "^16.9.0 || ^17.0.0"
+    "@types/react": "^16.9.0 || ^17.0.0 || ^18.0.0 || ^18.1.0",
+    "react": "^16.9.0 || ^17.0.0 || ^18.0.0 || ^18.1.0",
+    "react-dom": "^16.9.0 || ^17.0.0 || ^18.0.0 || ^18.1.0",
+    "react-test-renderer": "^16.9.0 || ^17.0.0 || ^18.0.0 || ^18.1.0"
```

Comment:
I think these could just be `^16.9.0 || ^17.0.0 || ^18.0.0` as `^18.1.0` is covered by the `^` on `18.0.0`.